.github/workflows/ci.yml

```diff
@@ -298,7 +298,7 @@ jobs:
         run: |
           printf 'User-agent: *\nDisallow: /' > build/robots.txt
 
-      - name: Push docs
+      - name: Push website
         run: |
           aws s3 sync build "s3://${{ secrets.DOCS_S3_BUCKET }}"
```

Comment:
Can you add a ` --exclude ".git/*" --delete` here?

Reply:
We are just syncing the `build` dir and `.git` will be outside that when syncing.

```
.
├── .git
└── website
    ├── build  <-- this
    ├── docs
    └── src
```

Let me add `--delete` though.